packages/schematics/angular/migrations/migration-collection.json

```diff
@@ -29,6 +29,11 @@
       "version": "7.0.3",
       "factory": "./update-7/index#updateDevkitBuildNgPackagr",
       "description": "Update an Angular CLI project to version 7."
+    },
+    "migration-07": {
+      "version": "7.4.0",
```

Comment:
we aren't going to ship a 7.4
can you change this to 8?

Reply:
Definitely